mud_manager.c

```diff
@@ -1174,7 +1278,16 @@ cJSON* parse_mud_content (request_context* ctx, int manuf_index)
                      acllist[acl_index].ace[ace_index].action = 1;
                   }
              }
-             acllist[acl_index].ace_count++;
```

Comment:
Can you please add a comment here saying what is happening? I don't think this is part of 'Check the "actions"' code block.

Reply:
Sure.  FYI what is happening is that we need to ignore certain entries in the MUD file if we don't have the right information up above.  And so we reuse that ACE in the next line.